### PR TITLE
1.1 add roles on groups

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

File: ApiBundle/Transformer/GroupTransformer.php [hunk 1]
```diff
@@ -2,24 +2,29 @@
...
+class GroupTransformer extends AbstractSecurityCheckerAwareTransformer
 {
     protected $translationChoiceManager;
 
     /**
      * @param TranslationChoiceManager $translationChoiceManager
      */
-    public function __construct(TranslationChoiceManager $translationChoiceManager)
-    {
+    public function __construct(
```
Comment (itkg-ngilain) [inline ApiBundle/Transformer/GroupTransformer.php]: @ggrasset : update php doc

File: UserAdminBundle/DataFixtures/MongoDB/LoadUserData.php [hunk 2]
```diff
@@ -51,7 +54,7 @@ public function load(ObjectManager $manager)
      *
      * @return User
      */
-    protected function generate($name, $group)
+    protected function generate($name, $group = null)
```
Comment (itkg-alavieille) [inline UserAdminBundle/DataFixtures/MongoDB/LoadUserData.php]: update php doc with default value

**Comment by itkg-alavieille** [comment]
else phpdoc :+1:

**Comment by ggrasset** [comment]
@itkg-alavieille @itkg-ngilain phpdoc update

File: ApiBundle/Transformer/GroupTransformer.php [hunk 3]
```diff
@@ -2,24 +2,30 @@
...
 /**
  * Class GroupTransformer
  */
-class GroupTransformer extends AbstractTransformer
+class GroupTransformer extends AbstractSecurityCheckerAwareTransformer
 {
     protected $translationChoiceManager;
 
     /**
+     * @param AuthorizationCheckerInterface $authorizationChecker
```
Comment (itkg-alavieille) [inline ApiBundle/Transformer/GroupTransformer.php]: align comment php doc : 

``
/**
 * @param AuthorizationCheckerInterface $authorizationChecker
 * @param TranslationChoiceManager      $translationChoiceManager
*/

File: ApiBundle/Transformer/GroupTransformer.php [hunk 4]
```diff
@@ -49,20 +55,24 @@ public function transform($group)
...
+                'open_orchestra_api_group_delete',
+                array('groupId' => $group->getId())
+            ));
+        }
+        if ($this->authorizationChecker->isGranted(AdministrationPanelStrategy::ROLE_ACCESS_UPDATE_GROUP)) {
+            $facade->addLink('_self_form', $this->generateRoute(
+                'open_orchestra_backoffice_group_form',
+                array('groupId' => $group->getId())
+            ));
+        }
```
Comment (itkg-alavieille) [inline ApiBundle/Transformer/GroupTransformer.php]: psr2, add an empty before return

File: BackofficeBundle/Resources/translations/role.en.yml [hunk 5]
```diff
@@ -23,9 +23,12 @@ open_orchestra_role:
     role_access_restore: Restore an entity from trash can
     role_access_status: Manage publication statuses
     role_access_theme: Manage themes
-    role_access_group: Manage user groups
     role_access_site: Manage a web site
     role_access_create_site: Create a web site
     role_access_update_site: Update a web site
     role_access_delete_site: Delete a web site
+    role_access_group: Manage user groups
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/translations/role.en.yml]: Manage an user group

File: BackofficeBundle/Resources/translations/role.fr.yml [hunk 6]
```diff
@@ -24,6 +24,9 @@ open_orchestra_role:
     role_access_status: Gérer les états de publication
     role_access_theme: Gérer les thèmes
     role_access_group: Gérer les groupes utilisateurs
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/translations/role.fr.yml]: Gérer un groupe utilisateur

**Comment by itkg-ngilain** [comment]
:+1: when the modification asked by itkg-alavieille is done

File: UserAdminBundle/DataFixtures/MongoDB/LoadUserData.php [hunk 7]
```diff
@@ -42,16 +42,18 @@ public function load(ObjectManager $manager)
...
+        $manager->persist($userNoAccess);
+
         $manager->flush();
     }
 
     /**
-     * @param string $name
-     * @param string $group
-     *
+     * @param $name
```
Comment (itkg-alavieille) [inline UserAdminBundle/DataFixtures/MongoDB/LoadUserData.php]: the syntax is : 

```
/**
 * @param string       $name
 * @param string|null $group
*/
```

**Comment by ggrasset** [comment]
@itkg-ngilain @itkg-alavieille Updates done until plural forms for global access rights.

File: UserAdminBundle/DataFixtures/MongoDB/LoadUserData.php [hunk 8]
```diff
@@ -42,16 +42,18 @@ public function load(ObjectManager $manager)
...
+        $manager->persist($userNoAccess);
+
         $manager->flush();
     }
 
     /**
      * @param string $name
-     * @param string $group
-     *
+     * @param string|null $group
```
Comment (itkg-alavieille) [inline UserAdminBundle/DataFixtures/MongoDB/LoadUserData.php]: align parameter

**Comment by itkg-alavieille** [comment]
else align parameter :

**Comment by ggrasset** [comment]
@itkg-alavieille Alignement done.

**Pull request merged**